website/www/site/static/js/copy-to-clipboard.js

```diff
@@ -0,0 +1,42 @@
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+$(document).ready(function() {
+    function copy() {
+        $(".copy").click(function(){
+            var text=$(this).siblings()[$(this).siblings().length-1].childNodes[0].innerHTML;
+            const el=document.createElement('textarea');
+            el.value=text;document.body.appendChild(el);
+            el.select();document.execCommand('copy');
+            document.body.removeChild(el);
+            alert('copied to clipboard');
+        })
+        $(".just-copy").click(function(){
+            var text=$(this).parent().siblings()[0].innerHTML;
+            const el=document.createElement('textarea');
+            el.value=text;document.body.appendChild(el);
+            el.select();document.execCommand('copy');
+            document.body.removeChild(el);
+            alert('copied to clipboard');
```

Comment:
This alert is strange to me. It's weird to have to click "OK" after copying.

Other implementations of this usually pop up a notification to let you know the content has been copied, but you don't have to acknowledge it. If it's too much to address this right now let's file a jira about it.

Reply:
From fastest ways I can change it to pop up under a button, but then there would be no popover when user hover on an icon

Reply:
Ok let's just file a jira for now